### PR TITLE
update API

### PR DESCRIPTION
**Pull request by casperdcl**
- use 'image_to_parametric_files' instead of 'image_to_parametric'
- add the use of a mask file, can automate the mask file selection according to the current image file organisation 
- change 'user_inputs', split linear_phase_start/end for logan_ref, logan_ref_k2p, mrtm, mrtm_k2p
- use 'models' (list of 'model's) instead of 'model'

closes https://github.com/JJiao/NiftyPAD/pull/2

### CODE REVIEW

File: niftypad/api/__init__.py [hunk 1]
```diff
@@ -7,15 +7,23 @@
...
 
 
-def kinetic_model(src, dst=None, params=None, model='srtmb_basis', input_interp_method='linear',
-                  w=None, r1=1, k2p=0.000250, beta_lim=None, n_beta=40, linear_phase_start=500,
-                  linear_phase_end=None, km_outputs=None, thr=0.1, fig=False):
+# def kinetic_model(src, dst=None, params=None, model='srtmb_basis', input_interp_method='linear',
+#                   w=None, r1=1, k2p=0.000250, beta_lim=None, n_beta=40, linear_phase_start=500,
+#                   linear_phase_end=None, km_outputs=None, thr=0.1, fig=False):
+
+def kinetic_models(src, dst=None, params=None, models=['srtmb_basis','logan_ref'], input_interp_method='linear',
```
Comment (casperdcl) [inline niftypad/api/__init__.py:14]: not sure about this @JJiao...

- why try to do multiple models in one function? Why not just call the function multiple times (once per model)?
- `linear_phase_(start|end)_(l|l2|m|m2)` seems very clunky (again, old API seems much cleaner)

**Comment by JJiao** [comment]
Hi Casper, I'm away this week, do you want to have a quick chat next week
about this?

File: niftypad/api/__init__.py (hunk 1 above)
Comment (casperdcl) [reply niftypad/api/__init__.py:14]: maybe better to do this:

```python
def kinetic_models(models=[...], **kwargs):
    results = []
    for model in models:
        results.append(kinetic_model(**kwargs))
    return results
```

Comment (JJiao) [reply niftypad/api/__init__.py:14]: Sure! Let's use 'kinetic_modelS'

**Comment by casperdcl** [comment]
I don't have push access to https://github.com/JJiao/NiftyPAD/tree/patch-3, so merged manually in e3f6221 instead

**Comment by casperdcl** [comment]
released in https://github.com/AMYPAD/NiftyPAD/releases/tag/v2.0.0

**Pull request closed without merging**